README.md

```diff
@@ -200,6 +199,26 @@ address found in the command above)
 docker-compose run --use-aliases -e FAKE_DNS=172.17.0.1 --service-ports boulder ./start.py
 ```
 
+Running tests without the `./test_dev.sh` wrapper:
+
+Run all unit tests
+
+```shell
+docker-compose run --use-aliases boulder go test -p 1 ./...
+```
+
+Run unit tests for a specific directory:
+
+```shell
+docker-compose run --use-aliases boulder go test $OPTARG
+```
+
+Run integration tests (omit `--pattern <REGEX>` to run all):
```

Comment:
You say `--pattern` here, but `--filter` below

Reply:
This is my fault for pushing my commit so late instead of leaving it for self-review in the morning.